src/templates/landing_pages/usb.jsx

```diff
@@ -0,0 +1,334 @@
+            'https://style.binary.com/binary.more.js',
+            'https://academy.binary.com/js/animate.min.js',
+            it.url_for('js/landing_pages/common.js'),
+            it.url_for('js/landing_pages/usb_page.js'),
+        ]}
+    >
+
+        <div id='page-top' />
+        <nav className='navbar navbar-custom navbar-fixed-top' role='navigation'>
+            <div className='navbar-container'>
+                <div className='nav-col-left'>
+                    <div className='logo-wrapper'>
+                        <a className='logo-parent logo-scroll' href='#page-top'>
+                            <div className='logo'>
+                                <div />
+                            </div>
+                            <div className='mobile-hide binary-logo-text'>
+                                <div />
+                            </div>
+                        </a>
+                    </div>
+                </div>
+                <div className='nav-col-right'>
+                    <div id='push' className='grill grill-ico' />
+                    <div id='push-nav' className='topnav'>
+                        <a className='page-scroll' href='#video'>{it.L('Overview')}</a>
+                        <a className='page-scroll' href='#why-usb'>{it.L('Why USB')}</a>
+                        <a className='page-scroll' href='#get-started'>{it.L('Get started')}</a>
+                        <a className='page-scroll' href='#benefits'>{it.L('Benefits')}</a>
+                        <a className='page-scroll' href='#tech'>{it.L('Technology')}</a>
+                        <a className='page-scroll' href='#faq'>{it.L('FAQ')}</a>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div className='home--header'>
+            <div className='header-content'>
+                <h1 className='header-title content-inverse-color ft-300'>{it.L('Welcome to the future')}</h1>
+                <span className='header-sub secondary-color'>{it.L('Introducing USB')}</span>
+                <span className='header-sub content-inverse-color'>{it.L('A stablecoin backed by the US Dollar')}</span>
+            </div>
+        </div>
+        <div className='section-md primary-bg-color coin-section'>
+            <div className='container center-text'>
+                <div className='gr-row align-items-center'>
+                    <div className='gr-2 gr-12-m gr-12-p'>
+                        <img className='usb-coin' src={it.url_for('images/usb_page/usb-coin-logo.png')} alt='USB' />
+                    </div>
+                    <div className='gr-10 gr-12-m gr-12-p'>
+                        <p className='content-inverse-color ft-400'>{it.L('Guaranteed by [_1] USB is a digital currency pegged to the US Dollar on a one-to-one basis. Send and receive payments with the peace of mind that your currency is backed by the strength of [_1] balance sheet.', it.broker_name)}</p>
```

Comment:
missing a comma after the sentence Guaranteed by Binary.com?: `Guaranteed by [_1], USB is a ...`